tfhe/src/core_crypto/entities/lwe_compact_ciphertext_list.rs

```diff
@@ -60,6 +61,12 @@ pub fn lwe_compact_ciphertext_list_size(
     mask_count.0 * lwe_dimension.0 + lwe_ciphertext_count.0
 }
 
+impl<Scalar: UnsignedInteger> LweCompactCiphertextList<Vec<Scalar>> {
+    pub fn get_mut_container(&mut self) -> &mut Vec<Scalar> {
+        &mut self.data
+    }
+}
+
```

Comment:
use as_mut() or as_ref()

Reply:
We need to mutably access the `Vec` in the tests to change its length and ensure it makes it non conformant

Reply:
then prefer a view ?

Reply:
not being able to modify a container in an entity is a conscious choice so that once you went through from_container you know the entity is valid

Reply:
Put behind #[cfg(test)]